examples/shaders/sky-shader/src/lib.rs

```diff
@@ -159,14 +160,29 @@ pub fn fs(constants: &ShaderConstants, frag_coord: Vec2) -> Vec4 {
 #[allow(unused_attributes)]
 #[spirv(fragment)]
 pub fn main_fs(
-    #[spirv(frag_coord)] in_frag_coord: Input<Vec4>,
+    #[spirv(frag_coord)] in_frag_coord: Input<Vec2>,
     constants: PushConstant<ShaderConstants>,
+    #[spirv(binding = 0)] img: UniformConstant<
+        Image<
+            { Dimensionality::TwoD },
+            { ImageDepth::No },
+            false,
+            false,
+            { Sampled::Yes },
+            { ImageFormat::Unknown },
+            { None },
+        >,
```

Comment:
I think we can't really have user facing code look like this, it's way to verbose so at a minimum we should have some typedefs for this.

Reply:
Thanks for adding the type aliasses @XAMPPRocky, we should use them here as well, code samples get copy & pasted far and wide.

Reply:
Yeah, I'll update it, I have to make the example work first. 😉